features/performance/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/openshift-kni/baremetal-deploy/performance
+
```

Comment:
Would it make sense to have this consume the vendor folder at `/performance` level?
Also, not sure how go modules would behave having a nested module.

Reply:
Dunno about nested modules, we'll see.
About consuming the `vendor` folder, that's up for discussion.

Reply:
Correct me if I am wrong, but my understanding from what we discussed offline yestarday was that everything should leverage the `/feature` level go.mod / vendor folder.

Reply:
Ah, sorry, I misunderstood your point. Then right, that's quoted aove a relic of a past version. I'll fix once your PR lands

Reply:
It's already in: https://github.com/openshift-kni/baremetal-deploy/pull/55